components/Heatmap.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import { View } from "react-native";
+import { ContributionGraph } from "react-native-chart-kit";
+
+export default function Heatmap({ style, dateDetails, width, height }) {
+  const getMonthLabel = (i) => {
```

Comment:
A better and shorter approach would be using an array of months instead of the whole switch case function. Something similar to : 
`const MonthLabels = (i) => { return ['Jan', 'Feb', 'Mar',.....][i] || 'June }`

Reply:
You're right. Should I change it? Or like eh

Reply:
That's more like a suggestions. Implement it if you feel like else we can merge this.